chatbot-core/data/preprocessing/preprocess_plugin_docs.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+from bs4 import BeautifulSoup
+from utils import (
+    remove_tags,
+    remove_html_comments,
+    get_visible_text_length
+)
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+INPUT_PATH = os.path.join(SCRIPT_DIR, "..", "raw", "plugin_docs.json")
+OUTPUT_PATH = os.path.join(SCRIPT_DIR, "..", "processed", "processed_plugin_docs.json")
+
+def main():
+    plugin_data = {}
+    with open(INPUT_PATH, "r", encoding='utf-8') as f:
+        plugin_data = json.load(f)
+
+    print(f"Handling {len(plugin_data.keys())} plugin docs.")
+    processed_plugin_docs = {}
+
+    for plugin_name, html_content in plugin_data.items():
```

Comment:
While the script is simple now, if this grows, extract the processing logic into a separate function